test/iphone-simulator.ts

```diff
@@ -164,4 +164,21 @@ describe("iphone-simulator", () => {
 			assert.ok(error.message.includes("Button \"NOT_A_BUTTON\" is not supported"));
 		}
 	});
+
+	it("should be able to get network info", async function() {
+		hasOneSimulator || this.skip();
+		const networkInfo = await simctl.getNetworkInfo();
+
+		// check that we get a valid network info object
+		assert.ok(typeof networkInfo === "object");
+		assert.ok(typeof networkInfo.type === "string");
+		assert.ok(typeof networkInfo.isConnected === "boolean");
+
+		// type should be one of the valid network types
+		const validTypes = ["wifi", "cellular", "none", "unknown"];
```

Comment:
shouldn't this test assert not only the type? cause these should be enforced by typescript. but maybe assert that all simulators return a specific type (is it wifi?)